crates/kv/Cargo.toml

```diff
@@ -15,7 +15,7 @@ slight-events-api = { path = "../events-api" }
 slight-runtime = { path = "../runtime" }
 slight-runtime-configs = { path = "../runtime-configs" }
 anyhow = "1"
-uuid = "1"
+uuid = { version = "1.1.2", features = ["v4"] }
```

Comment:
This seems to be a very strict version. Was the feature "v4" require it to be equal or above "1.1.2"?

Reply:
Actually, I think we have a bunch of these occurrences~ I'll make an issue to fix our dependencies all around.

Reply:
done #160 ✅